operator/internal/handlers/lokistack_create_or_update_test.go

```diff
@@ -357,10 +365,12 @@ func TestCreateOrUpdateLokiStack_WhenSetControllerRefInvalid_ContinueWithOtherOb
 		return nil
 	}
 
-	err := handlers.CreateOrUpdateLokiStack(context.TODO(), r, k, scheme, flags)
+	degraded, err := handlers.CreateOrUpdateLokiStack(context.TODO(), r, k, scheme, flags)
 
 	// make sure error is returned to re-trigger reconciliation
 	require.Error(t, err)
+	// make sure degraded is nil
+	require.Nil(t, degraded)
```

Comment:
Usually the contract is that when an error is returned, the result of the other values is undefined. Because of that I would not check for the value of `degraded` in this case.